modules/vectra.py

```diff
@@ -1612,7 +1612,9 @@ def post_stix_file(self, feed_id=None, stix_file=None):
         :param feed_id: id of threat feed (returned by get_feed_by_name)
         :param stix_file: stix filename
         """
-        return requests.post('{url}/threatFeeds/{id}'.format(url=self.url, id=feed_id), headers=self.headers,
+        headers = self.headers.copy()
+        del(headers['Content-Type'])
```

Comment:
I would recommend using headers.pop('Content-Type', None), which handles if the key is not present in the dict.

Reply:
I would also recommend using copy.deepcopy instead of dict.copy.  dict.copy is a shallow copy: https://docs.python.org/3.6/library/stdtypes.html#dict.copy

deepcopy handles copying nested dictionaries, so is preferred